pkg/maps/lbmap/lbmap.go

```diff
@@ -72,9 +73,15 @@ type UpsertServiceParams struct {
 // The corresponding backend entries (identified with the given backendIDs)
 // have to exist before calling the function.
 //
-// The service's prevActiveBackendCount denotes the count of previously active
+// The service's prevBackendCount denotes the count of previously stored
 // backend entries that were added to the BPF map so that the function can remove
 // obsolete ones.
+//
+// The service's non-active backends are appended to the active backends list,
+// and skipped from the service backends count so that the non-active backends
+// will not be considered for load-balancing traffic. The non-active backends
+// are, however, populated in the service map so that they can be restored
+// upon agent restart.
 func (lbmap *LBBPFMap) UpsertService(p *UpsertServiceParams) error {
```

Comment:
> The service's non-active backends are appended to the active backends list,
Why is this the case? 

Seems a bit odd to have non-active backends in the active backend list. 

If this is just for logging/cli dumping is it possible to dump the `NonActiveBackends` and omit the need to add them to the `ActiveBackends` set?

This would also reduce the code involved in separating the active/nonactive backends in service.go

Reply:
Got it